mitosheet/mitosheet/tests/test_transpile.py

```diff
@@ -702,16 +703,39 @@ def test_transpiled_with_export_to_xlsx_single():
     mito.code_options_update({'as_function': True, 'function_name': 'function', 'function_params': {'path': 'r"te' + "'" + 'st.xlsx"'}})
 
     assert "\n".join(mito.transpiled_code) == """from mitosheet.public.v3 import *
+import pandas as pd
 
 def function(df, path):
-    df.to_excel(path, sheet_name='df', index=False)
+    with pd.ExcelWriter(path, engine="openpyxl") as writer:
+        df.to_excel(writer, sheet_name="df", index=False)
     
     return df
 
 path = r"te'st.xlsx"
 
 df = function(df, path)"""
 
+def test_transpiled_with_export_to_xlsx_format():
```

Comment:
I think you can move this to the `test_export_to_file.py` file. We usually just test code chunks with their corresponding step.